lib/src/widgets/video_track_renderer.dart

```diff
@@ -52,10 +53,18 @@ class VideoTrackRenderer extends StatefulWidget {
 
 class _VideoTrackRendererState extends State<VideoTrackRenderer> {
   rtc.RTCVideoRenderer? _renderer;
+  bool _rendererReady = false;
```

Comment:
if only used for web, could we reflect in variable name? along with a comment it's web-only.